sqs/json_schemas/channel.json

```diff
@@ -47,6 +47,16 @@
           "description": "Is this a FIFO queue?",
           "default": false
         },
+        "deduplicationScope": {
+          "type": "string",
+          "description": "Specifies whether message deduplication occurs at the message group or queue level. Valid values are messageGroup and queue (default).",
+          "default": "queue"
+        },
+        "fifoThroughputLimit": {
```

Comment:
If these have a predefined values then it would be best to schematise them as enums.

Reply:
Agreed